app/main.js

```diff
@@ -5,6 +5,7 @@ var exported = {
   'child_process': null,
   'uuid': require('uuid'),
   'flowhub-registry': require('flowhub-registry'),
+  'fbp-protocol-client': require('fbp-protocol-client'),
```

Comment:
To be honest, that export there is just so we can load `fbp-protocol-client` in tests. We could do a separate webpack build for tests to remove it.